setup.py

```diff
@@ -49,7 +49,8 @@ def readme():
                       'plams@git+https://github.com/SCM-NV/PLAMS@master',
                       'pyparsing', 'pyyaml>=5.1', 'filelock'],
     extras_require={
-        'test': ['pytest', 'pytest-cov', 'pytest-mock', 'nbsphinx', 'assertionlib'],
+        'test': ['pytest', 'pytest-cov', 'pytest-mock', 'typing_extensions',
```

Comment:
`typing_extensions` is good but we should try to **nudge** our user to go for more modern Python version ;)

Reply:
True, but I feel this subject should be moved to an (eventual) discusion about dropping older python version.
Besides, some of the stuff in ``typing_extensions`` was only added to ``typing`` in 3.8.

Reply:
Agree